torch/lib/c10d/CMakeLists.txt

```diff
@@ -138,7 +138,7 @@ target_include_directories(c10d PRIVATE ${CMAKE_BINARY_DIR}/include)
 
 install(TARGETS c10d ARCHIVE DESTINATION lib)
 
-option(BUILD_EXAMPLES "Build examples" OFF)
+option(BUILD_EXAMPLES "Build examples" ON)
```

Comment:
I'm not sure if we really want to turn this on by default

Reply:
Just to make sure this code is always compiled, in case anyone changes it